GeothermalEnergy/origins.md

```diff
@@ -1,3 +1,35 @@
 # The origins of geothermal energy
 
-This chapter is about the origins/sources of geothermal energy
\ No newline at end of file
+This chapter is about the origins/sources of geothermal energy
+## What is geothermal energy (@ Wen)
+Geothermal energy, viewed as a promising clean and renewable energy source, is the heat stored in the rocks and fluids in the Earh. Technically, geothermal energy refers the heat contained in the upper crust with accessible depth. The total heat contained in the Earth is in the order of 12.6 x 10<sup>24</sup>, while in the crust, the order of 5.4 x 10<sup>21</sup> {cite}`WEC_Geothermal`.
+## Where is the heat from ? (@ Wen)
+The fact that there is heat stored in the Earth is typically evidenced by the intrusion of magma at volcanic sites and widespread distribution of hot springs. Actually, heat flux exists on the every squar meter of the Earth surface, which on average is 87 mW/m<sup>2</sup>, with a total (integrating over the whole surface) amount of ~ 4.4 x 10<sup>13</sup> J/s {cite}`heatflux1, heatflux2`. The heat flux observed on the surface of the Earth is a result of conduction, convection and radiation from the inner part of the Earth. 
+
+Now we know that heat flux exists across each square meter of the Earth's surface, and is attributed to various mechanisms such as conduction, convection, and radiation arising from its interior. The next question is, what is the origin of the heat? Before answering this question, it is worthwhile to introduce briefly the structure of our planet.
+
+![Earth](../Origin_pictures/Earth_structure.png)
```

Comment:
There is something not working here. Check that the image is correct. I get this error:

```bash
reading sources... [100%] origins                                               
/Users/localadmin/Dropbox/GitHub/Deliverable_4_2/GeothermalEnergy/origins.md:11: WARNING: image file not readable: ../Origin_pictures/Earth_structure.jpg
/Users/localadmin/Dropbox/GitHub/Deliverable_4_2/GeothermalEnergy/markdown.md:49: WARNING: missing institution in WEC_Geothermal
```

Reply:
thanks. Line 11 the format of figure was already changed to .JPG... In your branch is still .PNG?

Reply:
Can you create a new pull request with your fix?